### ISSUE TITLE
IdsDropDown: Accessibility  WCAG Guideline: 4.1.2 Name, Role, Value Defects

### ISSUE THREAD

**Issue by CindyMercadoReyes** [infor-design/enterprise-wc#1231]
**Describe the bug**
Issue 1: [Global] Edit fields role is unnecessarily provided for combo box.
Issue 2: [Global] Expanded/ collapsed state of combo boxes are not defined.

**To Reproduce**

Issue1: [Global] Edit fields role is unnecessarily provided for combo box.
WCAG Guideline: 4.1.2 Name, Role, Value

1. Launch the URL: https://main.wc.design.infor.com/ids-dropdown/example.html
2. Navigate to combo boxes using arrow keys and verify
3. Edit fields role is unnecessarily provided for combo box.

**Expected behavior**
Edit fields role should not be provided for combo box.

User Impact: 
If edit field role will be defined then it will confuse the screen reader users.

**Screenshots**
https://www.screencast.com/t/9ExDqnXaa

Dev Recommendation: Remove edit field role and provide combobox role for the element.

Issue 2: [Global] Expanded/ collapsed state of combo boxes are not defined.
WCAG Guideline: 4.1.2 Name, Role, Value

"1. Launch the URL: https://main.wc.design.infor.com/ids-dropdown/example.html
2. Navigate to combo boxes using arrow keys and verify"
3. Expanded/ collapsed state of combo boxes are not defined.

Expected Results:
Expanded/ collapsed state of combo boxes should be defined.

User Impact:
If expanded/ collapsed state will not be defined then it will confuse the screen reader users.

***Screenshots***
https://www.screencast.com/t/srMDRQkgii

Dev Recommendation: Provide aria expanded attributes for the combo box.

**Platform**
Win 10/ Chrome/ NVDA

**Additional context**
None

**Comment by tmcconechy** [comment]
Lets check this agains https://www.w3.org/TR/2017/NOTE-wai-aria-practices-1.1-20171214/examples/listbox/listbox-collapsible.html or other accessible examples. I wasnt sure if aria-expanded is used but it seems it is. So this seems valid.

But overall need to reconfirm the dropdown with aria. At the time i wrote it was good.
